deploy/cephfs/helm/templates/nodeplugin-daemonset.yaml

```diff
@@ -73,8 +73,6 @@ spec:
             - "--metadatastorage=k8s_configmap"
             - "--mountcachedir=/mount-cache-dir"
           env:
-            - name: HOST_ROOTFS
```

Comment:
why this is removed? please add a comment it will help in review

Reply:
Code prior to this was using this env variable to set hostRootFs here https://github.com/ceph/ceph-csi/blob/02bcb5f16ad062fd2b5814d866daad44c7ace738/pkg/rbd/rbd_attach.go#L34-L50 (in rbd)

The above variable is used to inspect `<hostRootFs>/proc` which is no longer needed as that function is removed.

CephFS is not using this variable at all, and was present superfluously, hence removed.